src/login/redux/sagas/getErrorsRequest.js

```diff
@@ -0,0 +1,26 @@
+import { put, call } from "redux-saga/effects";
+import { checkStatus, getRequest } from "sagas/common";
+import * as actions from "../actions/errorsMainActions";
+
+export function* requestConfig() {
```

Comment:
no, we are not using any config, when the page is loading it will update “available_language” and "static_faq_url". 
this will enable the toggle translation.

Reply:
Next question, do you want config? We can set up the jsconfig backend for to serve it.

Reply:
On second thought let's get a first version done before adding config.